disnake/poll.py

```diff
@@ -0,0 +1,444 @@
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+from datetime import timedelta
+from typing import TYPE_CHECKING, List, Optional, Union
+
+from . import utils
+from .emoji import Emoji
+from .enums import PollLayoutType, try_enum
+from .partial_emoji import PartialEmoji
+from .user import User
+
+if TYPE_CHECKING:
+    from datetime import datetime
+
+    from .abc import MessageableChannel
+    from .message import Message
+    from .state import ConnectionState
+    from .types.poll import (
+        Poll as PollPayload,
+        PollAnswer as PollAnswerPayload,
+        PollAnswerCount as PollAnswerCountPayload,
+        PollCreateAnswerPayload,
+        PollCreateMediaPayload,
+        PollCreatePayload,
+        PollMedia as PollMediaPayload,
+        PollResult as PollResultPayload,
+    )
+
+__all__ = (
+    "PollResult",
+    "PollAnswerCount",
+    "PollMedia",
+    "PollAnswer",
+    "Poll",
+)
+
+
+class PollAnswerCount:
+    """Represents a poll answer count from discord.
+
+    Attributes
+    ----------
+    id: :class:`int`
+        The ID of the answer the counts are of.
+    count: :class:`int`
+        The number of votes for this answer.
+    me_voted: :class:`bool`
+        Whether the current user voted for this answer.
+
+    .. versionadded:: 2.10
```

Comment:
versionadded should be above attributes/parameters, I don't think sphinx will add it to the description otherwise